dataactbroker/handlers/fileHandler.py

```diff
@@ -625,9 +625,9 @@ def publish_fabs_submission(submission):
         }
         logger.info(log_data)
 
-        # set publish_status to "publishing"
+        # set publish_status to 'publishing'
```

Comment:
I'm not sure but I thought we used double quotes in comments since they're more like English than Python and in English it'd be a double quote?

Reply:
I can't remember. I can change them back.